lti_consumer/tests/unit/test_lti_consumer.py

```diff
@@ -0,0 +1,665 @@
+"""
+Unit tests for LtiConsumerXBlock
+"""
+
+import unittest
+
+from datetime import datetime, timedelta
+from mock import Mock, PropertyMock, patch
+
+from lti_consumer.tests.unit.test_utils import make_xblock, make_request
+
+from lti_consumer.lti_consumer import LtiConsumerXBlock, parse_handler_suffix
+from lti_consumer.exceptions import LtiError
+
+
+USER_ID = '4rtkx56'
+HTML_PROBLEM_PROGRESS = '<div class="problem-progress">'
+HTML_ERROR_MESSAGE = '<h3 class="error_message">'
+HTML_LAUNCH_BUTTON = 'btn-lti-launch'
+HTML_IFRAME = '<iframe'
+HTML_SCRIPT = '<script type="application/javascript">'
+
+
+class TestLtiConsumerXBlock(unittest.TestCase):
+    """
+    Unit tests for LtiConsumerXBlock.max_score()
+    """
+
+    def setUp(self):
+        super(TestLtiConsumerXBlock, self).setUp()
+        self.xblock_attributes = {
+            'launch_url': 'http://localhost:8005/lti_launch',
```

Comment:
But, this isn't actually being used as a running process during tests, is it? I wouldn't expect a running xblock to be necessary for the unit tests...